newsroom/wire/views.py

```diff
@@ -108,7 +108,7 @@ def get_view_data() -> Dict:
             for f in app.download_formatters.values()
             if "wire" in f["types"]
         ],
-        "navigations": get_navigations_by_company(company, product_type="wire") if company else [],
+        "navigations": get_navigations_by_user(user, product_type="wire") if user else [],
```

Comment:
we need to get user product navigations + navigations for company products which have `0` seats so they are available for all users